shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -285,3 +295,22 @@ fun String.extractShortTypeFromExpanded(): String =
      * TODO do a clean implementation using info from @context
      */
     this.substringAfterLast("/").substringAfterLast("#")
+
+fun Map<String, Any>.toKeyValues(): Map<String, Any> {
+    return this.map { (key, value) -> key to simplifyValueIfNeeded(value) }.toMap()
+}
+
+private fun simplifyValueIfNeeded(value: Any): Any {
```

Comment:
Maybe `simplifyValueRepresentation` to be more aligned with wording in the spec?

Reply:
Or just `simplifyRepresentation`

Reply:
ok